### PR TITLE
Milestone 4: Portfolio, desktop version.

### PR DESCRIPTION
**Pull request by Jrmcuervo**
### Build portfolio desktop version of template 1.

In this milestone I:
- Finish the desktop version using media queries.
- Use grid and flexbox.
- Make sure that there are no linter errors.
- Make sure to follow all the specifications of the project.

### CODE REVIEW

File: idex.html [hunk 1]
```diff
@@ -11,208 +11,230 @@
...
+                    </div>
                 </li>
+                <hr>
                 <li>
-                    <label for=""></label>
-                    <textarea class="footer__input footer__textarea" id="msg" name="user_message"
-                        placeholder="Write your message here" maxlength="500" required></textarea>
+                    <div class="knowledge__titles">Skills<span class="material-symbols-outlined">
+                            chevron_right
+                        </span>
```
Comment (ichala) [inline idex.html:202]: - [ ] kindly align the image and the title of the languages to the left to match the figma design

image

figma: 

image

File: idex.html [hunk 2]
```diff
@@ -11,208 +11,230 @@
...
+            <div>
+                <h3 class="about__title">About <br> Myself</h3>
+                <p class="about__paragraph">Hello I’m a software developer! I can help you build a product , feature or
+                    website
+                    Look through some of my work and experience! If you like what you see and have a project you need
+                    coded,
+                    don’t hestiate to contact me.</p>
+                <h2 class="about__subtitle">LET'S CONNECT</h2>
+                <ul class="about__icons">
+                    <li><img src="./icons/github.svg" alt="github icon"></li>
```
Comment (ichala) [inline idex.html:172]: - [ ] kindly add the top right border-radius for ur about section to match the Figma design
image


Figma:
image

File: idex.html [hunk 3]
```diff
@@ -11,208 +11,230 @@
...
+                    </li>
+                    <li>
+                        <label for=""></label>
+                        <textarea class="footer__input footer__textarea" id="msg" name="user_message"
+                            placeholder="Write your message here" maxlength="500" required></textarea>
+                    </li>
+                </ul>
+                <button class="footer__button" type="submit" href="">Get In Touch</button>
+            </form>
+        </footer>
```
Comment (ichala) [inline idex.html:236]: - [ ] kindly update the background image for the contact-me section to match Figma design


image


Figma 
image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Great design ✔️


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
Hi @Jrmcuervo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**